api/config/packages/sentry.yaml

```diff
@@ -3,8 +3,9 @@ sentry:
     register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
     options:
         environment: '%env(SENTRY_ENVIRONMENT)%'
-        integrations:
-            - 'Sentry\Integration\IgnoreErrorsIntegration'
+        ignore_exceptions:
+            - 'Symfony\Component\ErrorHandler\Error\FatalError'
```

Comment:
It would be good to understand, why these should be ignored. To me it doesn't seem like an error that should be ignored. But there might be a good reason for it (maybe related to https://github.com/getsentry/sentry-symfony/issues/63?).

Reply:
See the changelog:

> The IgnoreErrorsIntegration integration was removed. Use the ignore_exceptions option instead.
> Previously, both Symfony\Component\ErrorHandler\Error\FatalError and 
> Symfony\Component\Debug\Exception\FatalErrorException were ignored by default.
> To continue ignoring these exceptions, make the following changes to the config file:
https://github.com/getsentry/sentry-symfony/blob/master/UPGRADE-5.0.md